MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,10 +1,170 @@
+    let text: String
+    let buttonText: String
+}
+
+// MARK: - Mock Data
+
+private let questions: [QuizQuestion] = [
+    QuizQuestion(
+        image: "The Godfather",
+        text: "Рейтинг этого фильма больше чем 6?",
+        correctAnswer: true),
+    QuizQuestion(
+        image: "The Dark Knight",
+        text: "Рейтинг этого фильма больше чем 6?",
+        correctAnswer: true),
+    QuizQuestion(
+        image: "Kill Bill",
+        text: "Рейтинг этого фильма больше чем 6?",
+        correctAnswer: true),
+    QuizQuestion(
+        image: "The Avengers",
+        text: "Рейтинг этого фильма больше чем 6?",
+        correctAnswer: true),
+    QuizQuestion(
+        image: "Deadpool",
+        text: "Рейтинг этого фильма больше чем 6?",
+        correctAnswer: true),
+    QuizQuestion(
+        image: "The Green Knight",
+        text: "Рейтинг этого фильма больше чем 6?",
+        correctAnswer: true),
+    QuizQuestion(
+        image: "Old",
+        text: "Рейтинг этого фильма больше чем 6?",
+        correctAnswer: false),
+    QuizQuestion(
+        image: "The Ice Age Adventures of Buck Wild",
+        text: "Рейтинг этого фильма больше чем 6?",
+        correctAnswer: false),
+    QuizQuestion(
+        image: "Tesla",
+        text: "Рейтинг этого фильма больше чем 6?",
+        correctAnswer: false),
+    QuizQuestion(
+        image: "Vivarium",
+        text: "Рейтинг этого фильма больше чем 6?",
+        correctAnswer: false)
+]
+
+// MARK: - Lifecycle
+
 final class MovieQuizViewController: UIViewController {
-    // MARK: - Lifecycle
+    
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+          return .lightContent
+    }
+    
+    private var currentQuestionIndex = 0
+    private var correctAnswers: Int = 0
+
+    @IBOutlet private weak var counterLabel: UILabel!
+    @IBOutlet private weak var imageView: UIImageView!
+    @IBOutlet private weak var questionLabel: UILabel!
+    @IBOutlet private weak var noButton: UIButton!
+    @IBOutlet private weak var yesButton: UIButton!
+
     override func viewDidLoad() {
         super.viewDidLoad()
+        show(quiz: convert(model: questions[currentQuestionIndex]))
+    }
+    
+    private func show(quiz step: QuizStepVeiwModel) {
+        counterLabel.text = step.questionNumber
+        imageView.image = step.image
+        questionLabel.text = step.question
+    }
+
+    private func convert(model: QuizQuestion) -> QuizStepVeiwModel {
+        return QuizStepVeiwModel(
+            image: UIImage(named: model.image) ?? UIImage(),
+            question: model.text,
+            questionNumber: "\(currentQuestionIndex + 1)/\(questions.count)")
+    }
+    
+    private func show(quiz result: QuizResultsVeiwModel) {
+        let allert = UIAlertController(
+            title: result.title,
+            message: result.text,
+            preferredStyle: .alert)
+        let action = UIAlertAction(title: result.buttonText, style: .default) {_ in
```

Comment:
после фигурной скобки в данном случае нужен пробел
```suggestion
        let action = UIAlertAction(title: result.buttonText, style: .default) { _ in
```